vip-application/src/main/java/fr/insalyon/creatis/vip/application/client/bean/boutiquesTools/BoutiquesOutputFile.java

```diff
@@ -47,7 +49,10 @@ public class BoutiquesOutputFile implements IsSerializable {
     private boolean list;
     private boolean optional;
     private String commandLineFlag;
-
+    private Set<String> pathTemplateStrippedExtensions;
+    private boolean isStripExtension;
```

Comment:
This boolean is not necessary.
I prefer you to use a condition on `pathTemplateStrippedExtensions` being not null and not empty in the gasw.vm instead.